### ISSUE TITLE
[Bug] Inbuild MsBuild macro not recognized

### ISSUE THREAD

**Issue by JKamsker** [3F/MvsSln#32]
Hey,
hope you are well!

When loading an sln with ``new Sln(@"C:\Users\JK\source\repos\path\to\my.sln", SlnItems.All & ~SlnItems.ProjectDependencies)``

Following error occurs:
`Microsoft.Build.Exceptions.InvalidProjectFileException: 'The imported project "C:\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.  C:\Users\JK\source\repos\path\to\Client\Client.vcxproj'`

I think its following statement, which is responsible in the vcxproj: `  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />`

In vs, `VCTargetsPath` resolves to `C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VC\v160\` and not `C:\`

Is there any way for me to fix that issue or is it really a bug?

**Comment by 3F** [comment]
Hello @JKamsker

This is duplicate of #1 and it will be completely resolved only after #23 ( [IeXod](https://github.com/3F/IeXod) 0.1 beta is already rolled out )

Before planned 3.0 please consider the following solutions for 2.x 

* https://github.com/3F/MvsSln/wiki/Advanced-Features#custom-msbuild-resolver
* You could also try to use only specific [SlnItems](https://github.com/3F/MvsSln/blob/master/MvsSln/SlnItems.cs#L30) for your task because some processing does not require project handlers at all. Try it first.


> hope you are well!

Thank you! I am in the progress... hope so o_o

Let me know if you still have the problems, I can try something
https://twitter.com/GitHub3F/status/1320425255291265026

Thanks for using,
